dbt-tests-adapter/dbt/tests/adapter/concurrency/test_concurrency.py

```diff
@@ -317,16 +319,22 @@ def test_concurrency(self, project):
         rm_file(project.project_root, "seeds", "seed.csv")
         write_file(seeds__update_csv, project.project_root, "seeds", "seed.csv")
 
-        results, output = run_dbt_and_capture(["run"], expect_pass=False)
-        assert len(results) == 7
+        results = run_dbt(["run"], expect_pass=False)
+
```

Comment:
We can remove `len(results) == 7` here since that's implicitly checked via the `dict` comparison below. For example, if there is another status that's not included in `expected_statuses` that has non-zero occurrence, then `assert result_statuses == expected_statuses` would fail.